doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -362,24 +366,28 @@ You can use the following Kconfig options for fine-grained control over which dr
    * :ref:`nrf_security_drivers_oberon` does not support RSA key pair generation.
 
 
-Secure Hash configurations
-**************************
-
-To configure the Secure Hash algorithms, set one or more of the following Kconfig options:
+Hash configurations
+*******************
 
-+-----------------------+-----------------------------------------------+
-| Hash algorithm        | Configuration option                          |
-+=======================+===============================================+
-| SHA-1                 | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_1`   |
-+-----------------------+-----------------------------------------------+
-| SHA-224               | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_224` |
-+-----------------------+-----------------------------------------------+
-| SHA-256               | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_256` |
-+-----------------------+-----------------------------------------------+
-| SHA-384               | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_384` |
-+-----------------------+-----------------------------------------------+
-| SHA-512               | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_512` |
-+-----------------------+-----------------------------------------------+
+To configure the Hash algorithms, set one or more of the following Kconfig options:
+
++-----------------------+---------------------------------------------------+
+| Hash algorithm        | Configuration option                              |
++=======================+===================================================+
+| SHA-1                 | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_1`       |
++-----------------------+---------------------------------------------------+
+| SHA-224               | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_224`     |
++-----------------------+---------------------------------------------------+
+| SHA-256               | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_256`     |
++-----------------------+---------------------------------------------------+
+| SHA-384               | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_384`     |
++-----------------------+---------------------------------------------------+
+| SHA-512               | :kconfig:option:`CONFIG_PSA_WANT_ALG_SHA_512`     |
++-----------------------+---------------------------------------------------+
+| MD5                   | :kconfig:option:`CONFIG_PSA_WANT_ALG_MD5`         |
++-----------------------+---------------------------------------------------+
+| RIPEMD160             | :kconfig:option:`CONFIG_PSA_WANT_ALG_RIPEMD160`   |
++-----------------------+---------------------------------------------------+
 
 
 Secure Hash driver configurations
```

Comment:
Are we removing Secure for this section as well (from the heading and the introductory sentence)?

Reply:
Thank you, good point